Makefile.kokkos

```diff
@@ -401,13 +405,16 @@ ifeq ($(KOKKOS_INTERNAL_USE_ARCH_NVIDIA), 0)
 endif
 
 ifeq ($(KOKKOS_INTERNAL_USE_ARCH_NVIDIA), 1)
+  KOKKOS_INTERNAL_NVCC_PATH := $(shell which nvcc)
   ifeq ($(KOKKOS_INTERNAL_COMPILER_CLANG), 1)
-    KOKKOS_INTERNAL_NVCC_PATH := $(shell which nvcc)
     CUDA_PATH ?= $(KOKKOS_INTERNAL_NVCC_PATH:/bin/nvcc=)
     ifeq ($(KOKKOS_INTERNAL_USE_OPENMPTARGET), 1)
       KOKKOS_INTERNAL_OPENMPTARGET_FLAG := $(KOKKOS_INTERNAL_OPENMPTARGET_FLAG) --cuda-path=$(CUDA_PATH)
     endif
   endif
+  ifeq ($(KOKKOS_INTERNAL_COMPILER_NVHPC), 1)
+    CUDA_PATH ?= $(KOKKOS_INTERNAL_NVCC_PATH:/bin/nvcc=)
+  endif
```

Comment:
Why is this necessary? Don't we set `KOKKOS_INTERNAL_NVCC_PATH` in the same way already in line 148?

Reply:
The line 148 is for the CUDA backend, which does not include the cases where other backends (OpenACC and OpenMPTarget) use NVIDIA GPUs.